source/includes/administration/_invoices.md

```diff
@@ -0,0 +1,147 @@
+## Invoices
+
+Note: Creating, deleting and updating invoices is not supported.
+
+<!-------------------- GET INVOICE -------------------->
+### Get customer invoices by reseller ID
+
+`GET /invoices/find/:reseller_id/customer_invoices`
+
+Retrieve a list of invoices belonging to the customers of the reseller specified.
+
+```shell
+# Retrieve customer invoices list
+curl "https://cloudmc_endpoint/rest/invoices/find/efd32752-c6f2-45cf-b494-cc6be8a45845/customer_invoices?includeAllSubOrgs=false&billingCycle=07-2021" \
+   -H "MC-Api-Key: your_api_key"
+```
+> The above command returns a JSON structured like this:
+
+```json
+{
+	"data": [{
+		"draftedDate": "2021-07-27T00:00:00Z",
+		"createdDate": "2021-07-27T00:00:00Z",
+		"status": "DRAFT",
+		"organization": {
+			"name": "org_name",
+			"id": "289ec5fb-0970-44e3-bca8-777a691e23c7"
+		},
+		"id": "20e9b8d8-b1cb-4462-b6e8-fbb8416b2cbb",
+		"detail": {
+			"discounted": true,
+			"discounts": [],
+			"startDate": "2021-07-01T00:00:00Z",
+			"endDate": "2021-08-01T00:00:00Z",
+			"currency": "CAD",
+			"subTotal": 272154.29,
+			"categories": [{
+				"discounted": true,
+				"unit": {
+					"unit": "HOUR",
+					"name": {}
+				},
+				"discounts": [],
+				"productId": "5bec5db2-b438-4610-ba73-f3ed50a7b26b",
+				"price": "40.0",
+				"name": {
+					"en": "memory",
+					"fr": "memory"
+				},
+				"subTotal": 65407.39,
+				"sku": "product_sku"
+			}]
+		}
+	}]
+}
+```
+
+Optional Query Parameters | &nbsp;
+---------- | -----------
+`includeAllSubOrgs`<br/>*boolean* | Whether or not to include sub-organizations of sub-organizations or just direct sub-organizations of the reseller.
+`billingCycle`<br/>*String* | The cycle which the invoice belongs to. Format is `MM/YYYY`
```

Comment:
The **billing** cycle 
 Format is `MM-YYYY`

Reply:
sorry about that😅 489d675